scripts/download_elevation_grid.py

```
@@ -0,0 +1,484 @@
+#!/usr/bin/env python3
+
+import asyncio
+import argparse
+import os
+from pathlib import Path
+import pandas as pd
+import numpy as np
+from datetime import datetime
+import logging
+from typing import Tuple, List, Dict
```

Ruff (0.12.2)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)